mlu_op.h

```diff
@@ -8755,6 +8755,225 @@ mluOpActiveRotatedFilterForward(const mluOpHandle_t handle,
+           const mluOpTensorDescriptor_t velocity_desc,
+           void *velocity,
+           const mluOpTensorDescriptor_t grad_desc,
+           void *grad,
+           const float lr,
+           const float beta1,
+           const float beta2,
+           const float bias1,
+           const float bias2,
+           const float epsilon);
+
+// Group: AdamW
+/*!
+ * @brief Creates a descriptor pointed by \p adamw_desc for AdamW operation. 
+ * The information is defined in ::mluOpAdamwDescriptor_t.
+ * For more information about descriptor, see "Cambricon MLU-OPS User Guide".
+ *
+ * @param[out] adamw_desc
+ * A host pointer to the AdamW descriptor that holds information about the
+ * AdamW operation.
+ *
+ * @par Return
+ * - ::MLUOP_STATUS_SUCCESS, ::MLUOP_STATUS_ALLOC_FAILED
+ *
+ * @par API Dependency
+ * - After calling this function, call ::mluOpSetAdamwDescAttr function to initialize
+ *   and set the information to the AdamW descriptor.
+ *
+ * @par Note
+ * - None.
+ *
+ * @par Requirements
+ * - None.
+ *
+ * @par Example
+ * - None.
+ *
+ * @par Reference
+ * - None.
+ */
+mluOpStatus_t MLUOP_WIN_API
+mluOpCreateAdamWDescriptor(mluOpAdamWDescriptor_t *desc);
+
+// Group: AdamW
+/*!
+ * @brief Initializes the descriptor \b adamw_desc that was previously created with
+ * ::mluOpCreateAdamWDescriptor function, and sets AdamW information
+ * to the descriptor \b adamw_desc. The information includes \b weight_decay , \b grad_scale 
+ * and \b use_nesterov for AdamW operation.
+ *
+ * @param[in] adamw_desc
+ * The descriptor of the RoiAlign operation. For detailed information,
+ * see ::mluOpAdamWDescriptor_t.
+ * @param[in] weight_decay
+ * The value of this parameter should be greater than 0.
+ * @param[in] grad_scale
+ * The value of this parameter should be greater than 0.
+ * @param[in] use_nesterov
+ * The value of this parameter should be true or false.
+ *
+ * @par Return
+ * - ::MLUOP_STATUS_SUCCESS, ::MLUOP_STATUS_BAD_PARAM
+ *
+ * @par Data Type
+ * - None.
+ *
+ * @par Data Layout
+ * - None.
+ *
+ * @par Scale Limitation
+ * - None.
+ *
+ * @par API Dependency
+ * - This function should be called after ::mluOpCreateAdamWDescriptor.
+ *
+ * @par Note
+ * - None.
+ *
+ * @par Example
+ * - None.
+ *
+ * @par Reference
+ * - None.
+ */
+mluOpStatus_t MLUOP_WIN_API
+mluOpSetAdamWDescAttr(mluOpAdamWDescriptor_t adamw_desc,
+                      mluOpAdamWDescAttribute_t attr,
+                      const void *buf,
+                      const size_t size_in_bytes);
+
+// Group: DeformConv
+/*!
+ * @brief Destroys a AdamW descriptor \p adamw_desc that was previously created by
+ * ::mluOpCreateAdamWDescriptor.
+ *
+ * @param[in] adamw_desc
+ *   The AdamW descriptor to be destroyed.
+ * @par Return
+ * - ::MLUOP_STATUS_SUCCESS, ::MLUOP_STATUS_BAD_PARAM
+ *
+ * @par Note
+ * - Call this function after calling the ::mluOpAdamW.
+ * - It is necessary to call this function destroy the AdamW descriptor to avoid the memory leaks.
```

Comment:
call this function destroy the AdamW descriptor to avoid the memory leaks》》
call this function to destroy the AdamW descriptor to avoid memory leak